mri/operators/fourier/utils.py

```diff
@@ -245,7 +245,7 @@ def generate_operators(data, wavelet_name, samples, mu=1e-06, nb_scales=4,
     return gradient_op, linear_op, prox_op, cost_op
 
 
-def get_stacks_fourier(kspace_loc):
+def get_stacks_fourier(kspace_loc, shape):
```

Comment:
I agree, however, there is a lot of dependencies for this function, and a change of variable name would lead to change multiples calls in the examples and test scripts. It is best if it stays as "shape" for now.

Reply:
You can use refactor in pycharm for this, its good practice to use the right names from start, can we please get this?

Reply:
I know, the problem is that it will affect some functions and scripts not related to this PR. I think this is a typo problem that needs to be fixed in a special PR to improve the code, make it user-friendly. I raised an issue #61

Reply:
I am sorry but I dont understand. We are not referring to all the refactoring mentioned in #61 . This PR adds new parameter shape. Now we would prefer if the code enters with right nomenclature directly. Other name changes can be done in another PR. But this particular one looks to me like is added in this PR and it only makes sense to add the right code in first commit than fix it up later.

Reply:
I am sorry but I dont understand. We are not referring to all the refactoring mentioned in #61 . This PR adds new parameter shape. Now we would prefer if the code enters with right nomenclature directly. Other name changes can be done in another PR. But this particular one looks to me like is added in this PR and it only makes sense to add the right code in first commit than fix it up later.

Reply:
@mathrip, I am not sure I understand your point. You introduced this variable so it means that everywhere the keyword argument `shape` is used for this function is because you introduced it, or am I mistaken?

Reply:
Yes you are right. So I changed it in this function because I had this argument. But it exists also in other functions such as extract_sensitivity, and the class Stacked3DNFFT. I thought you wanted me to change the name here as well... but I was mistaken !